ReactLearning/src/components/Carousel/index.js

```diff
@@ -0,0 +1,77 @@
+import React,{ Component } from 'react';
+import CarouselProducts from '../carouselProducts';
+import './Carousel.css';
+
+class Carousel extends Component {
+    constructor(props) {
+        super(props);
+        console.log("constructor - [Carousel Component]");
```

Comment:
remove console from final code

Reply:
Including all lifevycle methods was a part of assignment to check how all method works, Thats why all life cycle methods are included.